tools/screenshots/mermaid/poc.yaml

```
@@ -0,0 +1,8 @@
+- output: ../../app/assets/images/transit-gateways.png
```

Let's put them in a specific directory so that they're easier to keep track of, and so that we don't dump a bunch of unsorted content directly into `images`. What about:
```suggestion
- output: ../../app/assets/images/mermaid/transit-gateways.png
```